### ISSUE TITLE
Support for Tableau 9.3

### ISSUE THREAD

**Issue by adambalazs** [brilliant-data/tableau-dr#5]
Add support for a wider range of Tableau versions.
* [ ]  Build different versions of PG based on Tableau's version
* [ ]  Fix errors related to manifest.yml in tsbaks created by Tableau-DR

**Comment by adambalazs** [comment]
Tableau 9.2.4
postgres (PostgreSQL) 9.3.9
--enable-thread-safety --enable-integer-datetimes --enable-nls --with-ldap --with-ossp-uuid --with-libxml --with-libxslt --with-tcl --with-perl --with-python

Tableau 9.3.9
postgres (PostgreSQL) 9.3.13
--enable-thread-safety --enable-integer-datetimes --enable-nls --with-ldap --with-ossp-uuid --with-libxml --with-libxslt --with-tcl --with-perl --with-python

Tableau 10.1.1
postgres (PostgreSQL) 9.5.3
--enable-thread-safety --enable-integer-datetimes --enable-nls --with-ldap --with-openssl --with-ossp-uuid --with-libxml --with-libxslt --with-tcl --with-perl --with-python

Tableau 10.2 beta 1
postgres (PostgreSQL) 9.5.3
--enable-thread-safety --enable-integer-datetimes --enable-nls --with-ldap --with-openssl --with-ossp-uuid --with-libxml --with-libxslt --with-tcl --with-perl --with-python

Basically we need to implement a postgresql server compile strategy for 9.x and for 10.x according to the details above.
